src/featureslist.h

```diff
@@ -240,3 +240,11 @@ FEATURE_SIMPLE(gleanRust,             // Feature ID
                FeatureCallback_true,  // Can be flipped off
                QStringList(),         // feature dependencies
                FeatureCallback_false)
+
+FEATURE_SIMPLE(sentry,                     // Feature ID
+               "Sentry Crash Report SDK",  // Feature name
+               "2.13.0",                   // released
+               FeatureCallback_false,      // Can be flipped on
+               FeatureCallback_false,      // Can be flipped off
+               QStringList(),              // feature dependencies
+               FeatureCallback_inStaging)
```

Comment:
Wait, wasn't this only turned on for prod before? Did that change?

Reply:
The OG check was stage-only - given there might be need for UX changes and a legal check before we can release that

Reply:
Heh, nevermind me clearly remembering wrong.